lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -449,20 +447,20 @@ def _partition_pdf_sequenced(
                                 element.image_mode = cropped_image.mode
                                 element.image_size = cropped_image.size
                                 # print(element.properties)
-
         LogTime("finish", point=True)
         return deformable_layout
 
     def _partition_pdf_batched(
         self,
         file: BinaryIO,
         threshold: float = 0.4,
-        use_ocr=False,
-        ocr_images=False,
-        ocr_tables=False,
-        extract_table_structure=False,
+        use_ocr: bool = False,
```

Comment:
At some point after you get the paddle stuff in and you get the pdfminer stuff in, one of us should do a pass to clean up how we factor these methods and handle parameters. It's ridiculous how many times you have had to copy these changes, and we have seen that it makes it easy to make mistakes.

Reply:
Agreed.